docs/source/experiments/regression-robust04.md

```diff
@@ -0,0 +1,86 @@
+# PISA: Regression Tests for [Disks 4 & 5](https://trec.nist.gov/data_disks.html) (Robust04)
+
+## Indexing
+
+First, we will create a directory where all the indexes are going to be stored:
+
+```
+mkdir robust04
+```
+
+### Parsing
+
+```
+gzip -dc $(find /path/to/disk45/ -type f -name '*.*z' \( -path '*/disk4/fr94/[0-9]*/*' -o -path '*/disk4/ft/ft*' -o -path '*/disk5/fbis/fb*' -o -path '*/disk5/latimes/la*' \)) | bin/parse_collection -f trectext -b 10000 --stemmer porter2 --content-parser html -o robust04/fwd
```

Comment:
Can we make these listings more reader-friendly, i.e., break them into reasonable-length lines so that they fit in the text width?